conf/authdata.json

```diff
@@ -15,7 +15,7 @@
             "arn": "aws::iam:123456789013:bart",
             "keys": [{
                 "access": "USERBARTFUNACCESSKEY",
-                "secret": "verySecretKey1"
+                "secret": "verySecretKeyUser1"
```

Comment:
You are right - As I created a new user called Docker, I do not even need to change it. Thanks